src/ui/public/chrome/chrome.html

```diff
@@ -27,6 +30,15 @@
 
     <div class="bottom-apps">
       <div class="chrome-actions" kbn-chrome-append-nav-controls></div>
+
+      <!-- Open/close sidebar -->
+      <app-switcher-link
+        app-switcher-link-classes="appSwitcherButton.classes"
```

Comment:
I don't think it's necessary to add this custom attribute to the app-switcher-link directive. You can change this line to `class="{{appSwitcherButton.classes}}"`